specs/githubFetcher.spec.ts

```diff
@@ -0,0 +1,24 @@
+import axios from "axios";
+import { repositories } from "./fixtures/repositories";
+import { jest } from '@jest/globals';
+import { githubFetcher } from "../dataAccess/githubFetcher";
+
+describe('Github Fetcher', () => {
```

Comment:
I'm curious about these tests. The controller receives a RepoFetcher as a parameter, so we could inject a mocked Fetcher there because we only need the data inside a promise. Testing the github fetcher this way is a bit unnecessary as we will be testing axios, which is an external dependency 🤔. What do you think?

Reply:
You're right @CristopherDavid it doesn't provide much value because it's proving the Axios library, so I'm not longer keeping this test.